mne/viz/utils.py

```diff
@@ -2858,7 +2858,7 @@ def _plot_masked_image(ax, data, times, mask=None, yvals=None,
     else:
         # imshow for linear because the y ticks are nicer
         # and the masked areas look better
-        extent = [times[0], times[-1], yvals[0], yvals[-1] + 1]
+        extent = [times[0], times[-1], yvals[0], yvals[-1]]
```

Comment:
I think this is still wrong, I get:
![Figure_2](https://user-images.githubusercontent.com/2365790/66409901-30e74900-e9bf-11e9-90d3-97f64ae5806a.png)
```suggestion
        ds = np.array([np.median(np.diff(times)),
                       np.median(np.diff(yvals))]) / 2.
        extent = [times[0] - ds[0], times[-1] + ds[0],
                  yvals[0] - ds[1], yvals[-1] + ds[1]]
```
And later:
```
        time_lims = [extent[0], extent[1]]
        if ylim is None:
            ylim = [extent[2], extent[3]]
```
Gives:

![Figure_2](https://user-images.githubusercontent.com/2365790/66410490-2a0d0600-e9c0-11e9-8376-db1584e005ea.png)

Reply:
ha!  you beat me to it.  I was about to suggest the same solution.